### ISSUE TITLE
What hardware (development boards) does EOS support？

### ISSUE THREAD

**Issue by wanghq-thu** [EOS-team/EOS#2]
Will EOS support hardware such as Raspberry Pi or Orange Pi. If EOS supports specific hardware, it will be more convenient to use.

**Comment by ruiqurm** [comment]
Yes. One of the EOS target hardware is Raspberry Pi 4B. In November, an EOS LTS will be released, which allows you to burn in the SD card and use on the Raspberry Pi directly.
